RubeesEat.UnitTests/Model/InMemory/InMemoryPersonRepositoryTests.cs

```diff
@@ -69,25 +66,24 @@ public void GetCurrentUser_IsNotInDictionary_ThrowsException()
     }
 
     [Test]
-    public void TryGetPersonById_IsInDictionary_ReturnsTrueAndOutUser()
+    public void GetById_IsInDictionary_ReturnsUser()
     {
         Person defaultUser = new Person(Guid.NewGuid(), "DefaultFirstName", "DefaultLastName");
 
         var personRepository = new InMemoryPersonRepository(defaultUser.Id);
         personRepository.Add(defaultUser);
         
-        Assert.That(personRepository.TryGetById(defaultUser.Id, out var person), Is.True);
-        Assert.That(person, Is.EqualTo(defaultUser));
+        Assert.That(personRepository.GetById(defaultUser.Id), Is.EqualTo(defaultUser));
     }
     
     [Test]
-    public void TryGetPersonById_IsNotInDictionary_ReturnFalseAndOutNull()
+    public void TryGetPersonById_IsNotInDictionary_ReturnsNull()
     {
         Person defaultUser = new Person(Guid.NewGuid(), "DefaultFirstName", "DefaultLastName");
 
         var personRepository = new InMemoryPersonRepository(defaultUser.Id);
         
-        Assert.That(personRepository.TryGetById(defaultUser.Id, out var person), Is.False);
-        Assert.That(person, Is.Null);
+        Assert.That(personRepository.GetById(defaultUser.Id), Is.Null);
+
```

Comment:
```suggestion
        Assert.That(personRepository.GetById(defaultUser.Id), Is.Null);
```

Reply:
It looks like the suggestion is the same as our code. When i press "Commit suggestion" it also tells me "invalid because no changes were made to the code"
Do you mean we should put the asserts below each other, with no whitespaces?

Reply:
The idea is to remove the trailing empty line